Tests/Marketplace/marketplace_constants.py

```diff
@@ -54,6 +54,7 @@ class GCPConfig(object):
     BASE_PACK = "Base"  # base pack name
     INDEX_NAME = "index"  # main index folder name
     CORE_PACK_FILE_NAME = "corepacks.json"  # core packs file name
+    BUILD_BUCKET_PACKS_ROOT_PATH = 'content/builds/{}/{}/content/packs'
```

Comment:
please use keywords in the format so the constant's usage will be nicer to read
 
`BUILD_BUCKET_PACKS_ROOT_PATH.format(branch=branch_name, build=build_num`

```suggestion
    BUILD_BUCKET_PACKS_ROOT_PATH = 'content/builds/{branch}/{build}/content/packs'
```